content/doc/book/managing/nodes.adoc

```diff
@@ -45,9 +45,10 @@ clock time/sync and response time.
 A node is taken offline if any of these values
 go outside the configured threshold.
 
-The Jenkins controller itself runs on a special _built-in node_.
-It is possible to run agents and executors on this built-in node
-although this can degrade performance, reduce scalability of the Jenkins instance, and create serious security problems
+The Jenkins controller itself runs on a special _built-in node_. 
+The built-in node is a node inside the controller process. It is possible to run agents with it and it can have executors. 
```

Comment:
The first two sentences are directly contradicting each other.

The third sentence is wrong. The built-in node explicitly is _not_ an agent.